### PR TITLE
Install Singularity 3 from an official .deb, use newer ubuntu (jammy) on travis

### PR DESCRIPTION
**Pull request by bpoldrack**
Doesn't require changelog or release since it's only affecting the CI setup.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -42,8 +42,9 @@ before_install:
   # no recommends to avoid inheriting the entire multimedia stack
   - travis_retry sudo eatmydata apt-get install --no-install-recommends git-annex-standalone aria2 git-remote-gcrypt lsof gnupg nocache
   # chroot/container stuff
-  - travis_retry sudo eatmydata apt-get install xz-utils singularity-container
+  - travis_retry sudo eatmydata apt-get install xz-utils
```
Comment (bpoldrack) [inline .travis.yml:45]:
```suggestion
  - travis_retry sudo eatmydata apt-get install xz-utils jq
```

**Comment by bpoldrack** [comment]
Doesn't work on Travis yet, because there's no singularity release for `xenial`. Does anyone remember a specific reason to run that old an Ubuntu, @datalad/developers ?

**Comment by yarikoptic** [comment]
xenial??? that is nd16.04  , EOLed in April 2021...  we can't be using that!...  WTF  in https://app.travis-ci.com/github/datalad/datalad-container/jobs/590789811  says `distribution: debian/stretch/sid` and there is no error in installing singularity, only error in finding it or smth like that

```
 E               FileNotFoundError: [Errno 2] No such file or directory: 'singularity': 'singularity'
```

?  didn't check in further detail

**Comment by bpoldrack** [comment]
re xenial, @yarikoptic:
https://app.travis-ci.com/github/datalad/datalad-container/jobs/590789811#L9

That's why the construction of the URL to get the singularity release from, uses `xenial` (and gets a 404)

**Comment by bpoldrack** [comment]
When switching to `jammy` singularity installs fine.

However, now we get a new failure:

```
$ python setup.py build
Traceback (most recent call last):
  File "setup.py", line 16, in <module>
    cmdclass=cmdclass,
  File "/home/travis/virtualenv/python3.7.15/lib/python3.7/site-packages/setuptools/__init__.py", line 144, in setup
    return distutils.core.setup(**attrs)
  File "/opt/python/3.7.15/lib/python3.7/distutils/core.py", line 134, in setup
    ok = dist.parse_command_line()
  File "/opt/python/3.7.15/lib/python3.7/distutils/dist.py", line 483, in parse_command_line
    args = self._parse_command_opts(parser, args)
  File "/home/travis/virtualenv/python3.7.15/lib/python3.7/site-packages/setuptools/dist.py", line 925, in _parse_command_opts
    nargs = _Distribution._parse_command_opts(self, parser, args)
  File "/opt/python/3.7.15/lib/python3.7/distutils/dist.py", line 539, in _parse_command_opts
    cmd_class = self.get_command_class(command)
  File "/home/travis/virtualenv/python3.7.15/lib/python3.7/site-packages/setuptools/dist.py", line 764, in get_command_class
    self.cmdclass[command] = cmdclass = ep.load()
  File "/home/travis/virtualenv/python3.7.15/lib/python3.7/site-packages/pkg_resources/__init__.py", line 2450, in load
    return self.resolve()
  File "/home/travis/virtualenv/python3.7.15/lib/python3.7/site-packages/pkg_resources/__init__.py", line 2456, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
ModuleNotFoundError: No module named 'setuptools.command.build'
```

That's confusing me, though, since it's supposed to be python 3.7 all along. So, what changed here? Also: Do we even need to run `setup.py build`?

**Comment by yarikoptic** [comment]
yeah -- let's try without testing that. pushed

**Comment by yarikoptic** [comment]
> Could that be neurodebian being at fault here?

not sure how neurodebian could be at fault... if anything it is the python package at fault we are using in datalad wtf to figure out distribution.

**Comment by bpoldrack** [comment]
Alright, all green, let's go.

**Pull request merged**